Commit of 7 new Features

PhilTec-Philip opened the pull request:
@ColtG5 we already spoke about what my plan was.
So here are the new features:

- Added a Button to Force Stop Downloads
- Added a Button to decide whether a Monster should hear the played audio or not
- Added a Button to decide whether the current audio listening to should be looped or not
- Added a Button to Mute or Unmute audio (Client only)
- Added a Button to Fast-Forward and one for rewinding (10 seconds each click)
- Added a Button to toggle a Visual Effect on the Cart
- Configurable Keybind for opening the Boombox UI (only with the REPOConfig Mod)
- Edited the position of the 'Close' Button and enlarged it
- Replaced broken Discord Link in the Readme

I would be pleased to hear from you

best regards

PhilTec-Philip committed:
Readme-Update

PhilTec-Philip committed:
Updated Mod 1.2.0

Changelog
1.2.0
- Added a Button to Force Stop Downloads
- Added a Button to decide whether a Monster should hear the played audio or not
- Added a Button to decide whether the current audio listening to should be looped or not
- Added a Button to Mute or Unmute audio (Client only)
- Added a Button to Fast-Forward and one for rewinding (10 seconds each click)
- Added a Button to toggle a Visual Effect on the Cart
 - Configurable Keybind for opening the Boombox UI (only with the REPOConfig Mod)
- Edited the position of the 'Close' Button and enlarged it
- Replaced broken Discord Link in the Readme

PhilTec-Philip committed:
Updated Mod 1.2.1

## 1.2.1
 - Added an Audio Visualizer (+ UI Checkbox)

PhilTec-Philip commented:
**Update 1.2.1**
1 additional Feature:
- Added an Audio Visualizer (+ UI Checkbox)

Edit: When deactivating Checkbox the Cart disapeares, already doin' a fix

ColtG5 commented:
This looks incredible. Just tested it out and the new features are awesome, thank you so much for working on it! I'll leave you to work on a fix for the cart disappearing with that one checkbox. 

Meanwhile, I will probably look at allowing the new features you added to send networked events so every client gets synced updates of the track looping, fast-forwarding, forced downloaded clearing, etc., since it doesn't look like that is happening currently.

Once we have both of those things done, plus maybe some more tweaks and stuff, I can merge this and make a new 1.2.x release

PhilTec-Philip committed:
Updated Mod 1.2.2

**Update 1.2.2**
2 Fixes and one new Feature:
- Fixed a bug where the Cart despawned when unchecked Checkbox for the Audio Visualizer
- Fixed a bug where the Visualizer Blocks didn't despawn when unchecked Checkbox for the Audio Visualizer
- Added a "Recently played" History (UI)
